ProcessLib/ThermoMechanics/ThermoMechanicsProcessData.h

```diff
@@ -42,14 +42,16 @@ struct ThermoMechanicsProcessData
             linear_thermal_expansion_coefficient_,
         ParameterLib::Parameter<double> const& specific_heat_capacity_,
         ParameterLib::Parameter<double> const& thermal_conductivity_,
-        Eigen::Matrix<double, DisplacementDim, 1> const& specific_body_force_)
+        Eigen::Matrix<double, DisplacementDim, 1> const& specific_body_force_,
+        ParameterLib::Parameter<double> const* const nonequilibrium_stress_)
         : material_ids(material_ids_),
           solid_materials{std::move(solid_materials_)},
           reference_solid_density(reference_solid_density_),
           linear_thermal_expansion_coefficient(
               linear_thermal_expansion_coefficient_),
           specific_heat_capacity(specific_heat_capacity_),
           thermal_conductivity(thermal_conductivity_),
+          nonequilibrium_stress(nonequilibrium_stress_),
```

Comment:
Please `move nonequilibrium_stress(nonequilibrium_stress_)` after  `specific_body_force(specific_body_force_)`. The appearance order of members in the initializer list must be the same as that of members in the class declaration.

Reply:
ok, has been changed...